package.json

```diff
@@ -11,6 +11,7 @@
     "start-backend": "yarn workspace example-backend start",
     "start:next": "yarn workspace example-app-next start",
     "start-backend:next": "yarn workspace example-backend-next start",
+    "start:microsite": "cd microsite/ && yarn start",
```

Comment:
Think you can move this to `yarn --cwd microsite start` instead. It's a little cleaner.

Reply:
@benjdlambert that makes sense. However, I'm experiencing an issue where using ```startServerCommand: 'yarn --cwd microsite start'``` as the starting command within the ```microsite/lighthouserc.js``` configuration and ```yarn dlx @lhci/cli@0.11.x autorun --config microsite/lighthouserc.js``` to invoke this in the workflow still results in the main backstage example app being started up and not the documentation.